cohort-setup.md

```diff
@@ -1,10 +1,12 @@
 # How to set up a new cohort
 
-These are the steps required to set up a new cohort of subscriptions due to have their price increased.
+Previously, cohorts have been separated by product type, i.e.: Newspaper Vouchers and Newspaper Delivery.
```

Comment:
That was just for convenience.  Cohorts don't actually need to be by product type.
```suggestion
Previously, cohorts have been separated by product type, i.e.: Newspaper Vouchers and Newspaper Delivery.  The cohort names reflected this, eg. 'Vouchers2022'.  This was because the engine was being developed to support new product types while it was in use.  But cohorts don't need to be product-specific and the whole lot could be put in a single cohort.  How the cohorts are broken down depends on which subscriptions are ready for migration, either for business or technical reasons.
```

Reply:
Yep, thanks for making it clearer